plugins/woocommerce/client/blocks/tests/e2e/artifacts/storage-states/admin.json

```diff
@@ -0,0 +1 @@
+{"cookies":[{"name":"wordpress_test_cookie","value":"WP%20Cookie%20check","domain":"localhost","path":"/","expires":-1,"httpOnly":false,"secure":false,"sameSite":"Lax"},{"name":"wordpress_23778236db82f19306f247e20a353a99","value":"admin%7C1742572621%7C75fINM8gKLVfNSxXMQqlsIHhlj6Td75CleE3Wm1Liy8%7C18740c0e949620505ee20d8d87e16542225e6b7e1f457c08ff4d486a4682b8b5","domain":"localhost","path":"/wp-content/plugins","expires":-1,"httpOnly":true,"secure":false,"sameSite":"Lax"},{"name":"wordpress_23778236db82f19306f247e20a353a99","value":"admin%7C1742572621%7C75fINM8gKLVfNSxXMQqlsIHhlj6Td75CleE3Wm1Liy8%7C18740c0e949620505ee20d8d87e16542225e6b7e1f457c08ff4d486a4682b8b5","domain":"localhost","path":"/wp-admin","expires":-1,"httpOnly":true,"secure":false,"sameSite":"Lax"},{"name":"wordpress_logged_in_23778236db82f19306f247e20a353a99","value":"admin%7C1742572621%7C75fINM8gKLVfNSxXMQqlsIHhlj6Td75CleE3Wm1Liy8%7C3b5089bec4bb2eea7840fb12d895eb889ff7c7a0b23a1aba992dd472b37685bb","domain":"localhost","path":"/","expires":-1,"httpOnly":true,"secure":false,"sameSite":"Lax"},{"name":"wp-settings-time-1","value":"1742399821","domain":"localhost","path":"/","expires":1773935821.963,"httpOnly":false,"secure":false,"sameSite":"Lax"}],"nonce":"9849e59045","rootURL":"http://localhost:8889/wp-json/"}
```

Comment:
We should not be committing E2E tests artifacts. Curious why these weren't ignored

Reply:
I'm also not sure why those are added to begin with, as they werent part of the original PR but they came in the cherry pick, deleting them locally.